infra/nnfw/python/README.md

```diff
@@ -0,0 +1,46 @@
+# nnfwapi package
+
+`nnfwapi` is a package to run `nnpackage` with nnfw python API.
+
+This package includes nnfw python API module resulted from runtime build.
+
+It is provided separately according to each architecture(x86_64, armv7l, aarch64) from `nnfwapi/libnnfw_api_pybind.py` interface.
+
+## Requirement
+
+`nnfwapi` should be installed from PyPI.
+
+```
+$ pip install nnfwapi
```

Comment:
It does not work, maybe
```
$ pip install -i https://test.pypi.org/simple/ nnfwapi
```

I got this command from #11595.

Reply:
They tested using testpypi, not pypi.

Reply:
They tested using testpypi. We will have to upload it later to pypi due to account issues.
I think it is correct to write the document as above. Do you think it would be better to update this document later as well?

Reply:
As @ragmani said, we used TestPyPI for the test of the packaging process. In fact, we thought it should be uploaded to PyPI based on this case in the future. `README.md` was written in the form of 'pip install <package's name>' assuming that it will be uploaded to pypi. Perhaps, `pack.sh` written for convenient work may have been confused because the TestPyPI code is active instead of PyPI for uploading trhough `twine` .

Reply:
I was aware of your plan. But it does not work. I am not sure when it will be uploaded in official pypi. I think it needs to be updated and revisited before uploading official site. It is the reason I asked the working version. I didn't know how to install if #11595 did not exist.